### PR TITLE
Mesoshapes County Import - Round 2.2

### PR DESCRIPTION
**Pull request by stepps00**
This PR includes new, updated, and deprecated admin2 and admin1 features for:

- Armenia
  - The meso features are actually `localadmin` features and were imported accordingly. Since there were no existing features in the `localadmin` placetype in Armenia, this was a straight import of new features.
- Guinea
  - Meso features were imported as `county` features. Leftover records that did not map to meso features were deprecated.
- Malawi
  - These meso features are actually `region` features. There are no `county` level administrative divisions in Malawi, so the existing WOF `county` features were deprecated.
- Morocco
  - Meso features were imported as `county` features. Leftover records that did not map to meso features were deprecated.
- Zimbabwe
  - Meso features were imported as `county` features. Leftover records that did not map to meso features were deprecated.
- Ivory Coast
  - Meso features were imported as `county` features. Leftover records that did not map to meso features were deprecated.
- Namibia
  - Meso features were imported as `county` features. Leftover records that did not map to meso features were deprecated.
- Equatorial Guinea
  - Meso features were imported as `county` features. No existing features of that placetype were in WOF, so this country has all new `county features`.
- Kazakhstan
  - Meso features were imported as `county` features. Leftover records that did not map to meso features were deprecated.

### CODE REVIEW

**Comment by stepps00** [comment]
Part of https://github.com/whosonfirst-data/whosonfirst-data/issues/558.

File: data/856/750/07/85675007.geojson [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "wof:lang":[
         "eng"
     ],
-    "wof:lastmodified":1481825536,
+    "wof:lastmodified":1481827136,
```
Comment (nvkelso) [inline data/856/750/07/85675007.geojson]: Deprecated bunk admin2 in Malawi needs deprecated date and to be marked not current?

**Comment by stepps00** [comment]
Regarding https://github.com/whosonfirst-data/whosonfirst-data/pull/561#pullrequestreview-13656031:

Both Statoids and Wikipedia show 59 districts in Zimbabwe, but the country's Statistics site (http://www.zimstat.co.zw/) hosts data that include more that 59 features. As an example, the data from the government portal includes a feature for "Chipinge" and "Chipinge Urban", while Statoids and Wikipedia only call out "Chipinge." This is the cause of the import commit containing about two dozen extra features.

My thought is to import as-is, since these are shapes directly from the government statistics site, but I also see the case for dissolving and re-working the import commits for Zimbabwe.

**Comment by stepps00** [comment]
Regarding https://github.com/whosonfirst-data/whosonfirst-data/pull/561#pullrequestreview-13656789:

https://github.com/whosonfirst-data/whosonfirst-data/issues/603 was filed to investigate missing `admin2` features in Morocco.

**Comment by stepps00** [comment]
Regarding https://github.com/whosonfirst-data/whosonfirst-data/pull/561#pullrequestreview-13657568: 

The import file for `admin1` features in Malawi includes four "city" features that are separated from the region they fall within. This issue is being tracked in https://github.com/whosonfirst-data/whosonfirst-data/issues/604.

**Comment by stepps00** [comment]
Regarding https://github.com/whosonfirst-data/whosonfirst-data/pull/561#pullrequestreview-13654988:

https://github.com/whosonfirst-data/whosonfirst-data/issues/605 has been opened to track the mis-match between the Namibia import features and lists on Wikipedia and Statoids.

**Comment by stepps00** [comment]
Regarding https://github.com/whosonfirst-data/whosonfirst-data/pull/561#pullrequestreview-13654720:

https://github.com/whosonfirst-data/whosonfirst-data/issues/606 has been opened to track the discrepency in Equatorial Guinea.

**Comment by nvkelso** [comment]
Regarding https://github.com/whosonfirst-data/whosonfirst-data/pull/561#issuecomment-271431180 for Zimbabwe (Github, why you no thread!?):

Let's import as-is then. This sounds similar to Hungary where there is a difference of opinion if the "urban" parts and the "rural" parts are one-and-the_same, or different county features. In Hungary we went with "different", and in this case it's matches what the Zimbabwe gov't is advertising, so everyone else should catchup. (This will make our Statoids import challenging.)

**Comment by nvkelso** [comment]
Ready to merge, over to @thisisaaronland.

**Pull request merged**